packages/core/src/welcome/_storybook-styles.scss

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright IBM Corp. 2023, 2023
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../../ibm-products/src/global/styles/project-settings' as c4p-settings;
+@use '@carbon/styles/scss/colors';
+@use '@carbon/styles/scss/spacing' as *;
+@use '@carbon/styles/scss/type';
+
+.welcome__container {
+  position: fixed;
+  width: 100vw;
+  min-height: 100vh;
+  padding: $spacing-09;
+  background-attachment: fixed;
+  /* stylelint-disable-next-line carbon/theme-token-use */
+  background-color: colors.$purple-10;
+  background-position: center right;
```

Comment:
Changed it to `center left` earlier today — originally wanted more of Laura’s lovely banner to show but since we changed it to cover, I think this works better for smaller screens.